README.md

```diff
@@ -14,13 +14,79 @@ the AGW implements an evolved packet core (EPC), and a combination of an AAA and
 with existing, unmodified commercial radio hardware.<br>
 For more information on Magma please visit the [official website](https://magmacore.org/).
 
+> **WARNING:** Installing this snap will affect your computer's networking configuration.
+> Make sure it is installed on designated hardware (personal computers are strongly discouraged).
+
+### System requirements
+
+**Hardware (baremetal strongly recommended)**
+
+- Processor: x86-64 dual-core processor (around 2GHz clock speed or faster)
+- Memory: 4GB RAM
+- Storage: 32GB or greater SSD
+
+**Networking**
+
+- At least two ethernet interfaces (SGi and S1)
+- Internet connectivity from SGi interface
+
+**Operating System**
+
+- Ubuntu 20.04 LTS
+  ([Ubuntu installation guide](https://help.ubuntu.com/lts/installation-guide/amd64/index.html))
+
+
 ## Usage
 
-Please see
-[How-to: Deploy Magma Access Gateway using snap](/documentation/how-to-guides/deploy_magma_access_gateway.md).
+Unless specified otherwise, all actions in this guide are executed using `root` account.
```

Comment:
Is there a place where it is specified otherwise? If not let's start with the 'All actions...`

Reply:
Agreed.